docs/current_docs/quickstart/daggerize.mdx

```diff
@@ -27,6 +27,36 @@ git clone https://github.com/dagger/hello-dagger
 cd hello-dagger
 ```
 
+### Configure Dagger Cloud (optional)
+
+:::note
+This step is optional and will create a Dagger Cloud individual plan account. The individual plan is free of charge for a single user. You will require a GitHub account for account setup and identity verification. If you prefer not to sign up for Dagger Cloud, you can skip this section.
+:::
+
+Dagger Cloud is an online visualization tool for Dagger pipelines. It provides a web interface to visualize each step of your pipeline, drill down to detailed logs, understand how long operations took to run, and whether operations were cached.
+
+Create a new Dagger Cloud account by running `dagger login`:
+
+```shell
+dagger login
+```
+
+The Dagger CLI will invite you to authenticate your device by displaying a link containing a unique key. Click the link in your browser, and verify that you see the same key in the Dagger Cloud Web interface.
+
+```shell
+$ dagger login
+Browser opened to: https://auth.dagger.cloud/activate?user_code=FCNP-SRLM
+Confirmation code: FCNP-SRLM
+```
+
+Once you confirm your authentication code, your Dagger CLI will be authenticated and you will get redirected to your newly created Dagger Cloud organization.
+
+After successfully creating your organization, all future `dagger call` pipelines
+will be visualized in Dagger Cloud interface.
+
+![login](/img/cloud/hello-dagger.gif)
```

Comment:
This is in the wrong place, it should replace the FIXME further down.

Reply:
Two concerns about this image:
- Can it be edited so it doesn't say `marcos-test`? maybe something neutral like `test`, `dagger-test`, `myorg`, etc? Or just blur out the org name?
- The function is called `HelloDagger` but the trace image shows `helloDagger`, capitalization is different. Is that expected?

Reply:
> * Can it be edited so it doesn't say `marcos-test`? maybe something neutral like `test`, `dagger-test`, `myorg`, etc? Or just blur out the org name?

yes, I can change that



> * The function is called `HelloDagger` but the trace image shows `helloDagger`, capitalization is different. Is that expected?

yes. That's how cloud displays it